.github/workflows/ci.yml

```
@@ -0,0 +1,60 @@
+name: GitHub CI
+
+on:
+  push:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+      - '*.yml'
+  pull_request:
+    paths-ignore:
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.md'
+      - '*.yml'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout nxdk
+        uses: actions/checkout@v2
+        with:
+          repository: XboxDev/nxdk
+          submodules: recursive
+      - name: Set xbox_kernel_test_suite dir
+        id: xkts
+        run: echo "::set-output name=dir::projects/xbox_kernel_test_suite"
+      - name: Checkout xbox_kernel_test_suite
+        uses: actions/checkout@v2
+        with:
+          path: ${{ steps.xkts.outputs.dir }}
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y update
+          sudo apt-get -y install lld llvm
+      - name: Build
+        working-directory: ${{ steps.xkts.outputs.dir }}
+        run: make -j$(nproc)
+      - name: Set artifact+release parameters
```

```suggestion
      - name: Set artifact + release parameters
```
or 
```suggestion
      - name: Set artifact/release parameters
```
for readability reason